ticdc/troubleshoot-ticdc.md

```diff
@@ -396,7 +396,6 @@ If the result of `pd-ctl service-gc-safepoint --pd <pd-addrs>` does not have `gc
 
 - If your PD version is v4.0.8 or earlier, refer to [PD issue #3128](https://github.com/tikv/pd/issues/3128) for details.
 - If your PD is upgraded from v4.0.8 or an earlier version to a later version, refer to [PD issue #3366](https://github.com/tikv/pd/issues/3366) for details.
-- For other situations, report the execution result of the above command to our [Slack channel](https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=pingcap-community).
```

Comment:
Why remove this line? should we replace Slack with the TiDB Internals forum?

Reply:
[TiDB Internals Forum](https://internals.tidb.io/) is for TiDB development discussions. It seems to be a scenario when using TiDB here. So I didn't replace Slack with TiDB Internals. Is it appropriate to lead to asktug?

Reply:
> > [TiDB Internals Forum](https://internals.tidb.io/) is for TiDB development discussions. It seems to be a scenario when using TiDB here. So I didn't replace Slack with TiDB Internals. Is it appropriate to lead to asktug?
> 
> AskTUG is a Chinese website and cannot be placed in an English doc. Maybe we can still provide the internals link because what we want by using a link here is to provide a channel for users to ask for technical support.

Maybe you should ask @zhangyangyu if it is appropriate.

Reply:
internals are meant to provide a place for developing TiDB/TiKV/TiDB tools. I think it's not right to use it as a place to seek for tech support like askTUG. It's a known limitation we don't have a English counterpart of askTUG.

Reply:
Got it. Then I'm OK that we remove this part.@luzizhuo